packages/core/src/Tab/Tab.spec.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react'
+import { render, screen } from '../__test__/testing-library'
+import { userEvent } from '@storybook/testing-library'
+import { Tab } from './Tab'
+
+describe('Tab', () => {
+  const tabsContent = [
+    { id: 'tab1', children: <div>Tab 1 Content</div> },
+    { id: 'tab2', children: <div>Tab 2 Content</div> },
+  ]
+
+  const tabsData = [
+    { id: 'tab1', text: 'Tab 1' },
+    { id: 'tab2', text: 'Tab 2' },
+  ]
+
+  it('renders tabs with correct text', () => {
+    render(<Tab type='button' tabsContent={tabsContent} tabsData={tabsData} />)
+
+    expect(screen.getByText('Tab 1')).toBeInTheDocument()
+    expect(screen.getByText('Tab 2')).toBeInTheDocument()
```

Comment:
Could we please use `getByRole` queries here? Tabs need to be accessible by default.